### ISSUE TITLE
[Bug]: Timed Push Notifications are not received at the correct hour

### ISSUE THREAD

**Issue by westy92** [OneSignal/OneSignal-Android-SDK#1829]
### What happened?

Since upgrading my apps to OneSignal Flutter 5.0.0, my messages that are supposed to be delivered in each user's timezone at 7pm are being delivered hours early. My Android device in Central Time is receiving the notification at 2pm instead of 7pm.

### Steps to reproduce?

```Markdown
1. Send a timed notification for 7pm in the user's timezone.
```

### What did you expect to happen?

Notification should be received at 7pm. Instead it is received at 2pm.

### OneSignal Android SDK version

5.0.0

### Android version

12

### Specific Android models

```Markdown
All
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jennantilla** [comment]
Hello @westy92 thanks for reporting. We'll look into this and get back to you as soon as possible.

**Comment by emawby** [comment]
This appears to have 2 causes
1. The properties object is not initialized with the device's time zone
2. The properties object is not sent with the create user call (iOS sends language and timezone_id)
WiP branch [here](https://github.com/OneSignal/OneSignal-Android-SDK/tree/fix/timezone_id_not_set)

**Comment by emawby** [comment]
Fixed in #1842

**Comment by jennantilla** [comment]
@westy92 thank you for your patience! We have a fix for this in our latest release, [5.0.2](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/5.0.2). Please let us know if you have any additional questions or concerns!
